game/src/sandbox/mod.rs

```diff
@@ -444,7 +444,7 @@ impl AgentMeter {
                 .centered_vert(),
                 format!("{} trips captured", prettyprint_usize(n)).draw_text(ctx),
                 ctx.style()
-                    .btn_solid_text("Stop")
+                    .btn_solid_primary_text("Finish Capture")
```

Comment:
Took some liberties with the copy... "Stop" is sort of ambiguous as to whether it means "complete" or "cancel"